### PR TITLE
Design/new site

### PR DESCRIPTION
**Pull request by mtnbarreto**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1 +1,179 @@
...
+<div data-w-id="4d0974fd-4408-a21f-92c9-808ce69bb6d4" class="footer_section div-block-26 footer-margin">
+      <div class="footer">
+        <div class="container is-footer">
+          <div class="footer_top">
+            <div class="spacing is-small"></div><img src="https://uploads-ssl.webflow.com/5f2030e6ee0a9b09e8032b4d/5f513134a6dfed5850144026_xmartlabs-logo-white.svg" alt="" class="xl_footer_logo">
+            <div class="spacing is-small"></div>
+            <div class="footer_nav_left_section"><a href="https://xl4.webflow.io/work" class="is-big is-white is-footer-link">Work</a>
```
Comment (mlvieras) [inline _includes/footer.html:7]: We should probably change these links to our current web right?

File: _includes/head.html [hunk 2]
```diff
@@ -21,6 +21,6 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/fluidbox/1.4.3/jquery.fluidbox.min.js"></script>
 
 
-  <script src="/js/webflow.f19375dc7.js"></script>
+  <!-- <script src="/js/webflow.7892a673f.js"></script> -->
```
Comment (mlvieras) [inline _includes/head.html:24]: Maybe we should remove this

File: _includes/head_post.html [hunk 3]
```diff
@@ -31,15 +31,15 @@
...
+  <link rel="stylesheet" href="{{ "/css/xl4.webflow.0b1398f65.css" | relative_url }}">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fluidbox/1.4.3/css/fluidbox.css">
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url_blog }}">
   <meta name="Copyright" content="&copy; 2020 Xmartlabs" />
   <link rel="shortcut icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" />
 
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/fluidbox/1.4.3/jquery.fluidbox.min.js"></script>
-  <script src="/js/webflow.f19375dc7.js"></script>
+  <!-- <script src="/js/webflow.7892a673f.js"></script> -->
```
Comment (mlvieras) [inline _includes/head_post.html:42]: This one too

File: _sass/_bootstrap.scss [hunk 4]
```diff
@@ -637,8 +636,8 @@ pre code {
 .container {
   margin-right: auto;
   margin-left: auto;
-  padding-left: 15px;
-  padding-right: 15px;
+  // padding-left: 15px;
```
Comment (mlvieras) [inline _sass/_bootstrap.scss:639]: Commented code here

File: _includes/header.html [hunk 5]
```diff
@@ -1,17 +1,210 @@
 <header>
+<div data-collapse="medium" data-animation="default" data-duration="400" role="banner" class="nav w-nav">
+      <div class="container is-flex mobile-horizontal">
+        <div class="nav_menu">
+          <a title="Xmartlabs home" href="https://xl4.webflow.io" class="nav_logo w-inline-block" style="opacity: 1;">
```
Comment (eb16) [inline _includes/header.html:5]: Change it to:

`<a title="Xmartlabs home" href="https://xmartlabs.com" class="nav_logo w-inline-block" style="opacity: 1;">` instead.

(I don 't know why I don't have the `suggest changes` button)

File: _includes/header.html [hunk 6]
```diff
@@ -1,17 +1,210 @@
...
+          </a>
+          <a href="https://xl4.webflow.io/work" aria-current="page" class="nav_menu_item is-1 w-inline-block" style="transform: translate3d(0px, 0px, 0px) scale3d(1, 1, 1) rotateX(0deg) rotateY(0deg) rotateZ(0deg) skew(0deg, 0deg); transform-style: preserve-3d; opacity: 1;">
```
Comment (eb16) [inline _includes/header.html:8]: Same here and the links below

**Pull request merged**